client/sites/components/sites-dataviews/index.tsx

```diff
@@ -74,8 +85,10 @@ const DotcomSitesDataViews = ( {
 	// The current selection is a derived value which is [dataViewsState.selectedItem.ID].
 	// (See the `getSelection()` function below.)
 	const onSelectionChange = () => {};
-	const getSelection = ( dataViewsState: DataViewsState ) =>
-		dataViewsState.selectedItem ? [ dataViewsState.selectedItem.ID ] : undefined;
```

Comment:
> // However, when don't need to do anything in the callback, because we already maintain dataViewsState.selectedItem.

The above comment is no longer accurate now 😄

Reply:
Fixed